src/app.rs

```diff
@@ -237,7 +236,7 @@ where
     /// Registers contract code (like uploading wasm bytecode on a chain),
     /// so it can later be used to instantiate a contract.
     /// Only for wasm codes
-    pub fn store_wasm_code(&mut self, code: WasmContract) -> u64 {
+    pub fn store_wasm_code(&mut self, code: Vec<u8>) -> u64 {
```

Comment:
This allows for cw-multi-test handling of module caching... The interface is also simpler for users